features/admin.organizations.v1/pages/organization-edit.tsx

```diff
@@ -139,7 +139,8 @@ const OrganizationEditPage: FunctionComponent<OrganizationEditPagePropsInterface
         getOrganization(organizationId)
             .then((organization: OrganizationResponseInterface) => {
                 setOrganization(organization);
-                setFilterQuery("name eq " + organization?.name);
+                const escapedOrgName = organization?.name?.replace(/\\/g, '\\\\');
```

Comment:
Shall we use proper typing for the variables? and let's use double quotes for string literals.

```suggestion
                const escapedOrgName: string = organization?.name?.replace(/\\/g, "\\\\");
```